packages/router/src/lib/use-load.ts

```diff
@@ -0,0 +1,17 @@
+import { Injector, assertInInjectionContext, inject } from '@angular/core';
+import { ActivatedRoute, Data } from '@angular/router';
+import { map } from 'rxjs';
+import { PageServerLoad } from './route-types';
+
+export function useLoad<
```

Comment:
I'd use the `inject` prefix in the case of Angular, but that is just a preference 😅 

```suggestion
export function injectLoad<
```

Reply:
We already have `injectContent`, `injectContentFiles`, etc. So I'd also use the `inject` prefix because of the consistency.

Reply:
Good point. Let's rename them all 😅 

j/k